addons/mail/static/src/components/chat_window/chat_window.xml

```diff
@@ -24,7 +24,7 @@
                     <ThreadViewer
                         class="o_ChatWindow_thread"
                         composerAttachmentsDetailsMode="'card'"
-                        hasComposer="chatWindow.thread.model !== 'mail.box'"
+                        hasComposer="chatWindow.thread.model !== 'mail.box' and !chatWindow.thread.mass_mailing"
```

Comment:
Maybe it wasn't allowed before? But now that it is not linked to discuss, maybe we can keep the composer in this case now. Did you have any issue with it?

Reply:
It's not really usable as the composer takes too much place, maybe ask @alexkuhn what does he think about that

Reply:
I think it was hidden before our refactoring because this takes too much space and wasn't thought with a narrow thread viewer like in a chat window.

This adds to the batch of functional problems we have, so again let's be conservative and also hide it (no regression) until big functional changes in future (if any? let's be optimistic!).

And don't forget to still show it in mobile... Otherwise it's not possible to post messages in mailing channels in mobile at all! :o)

Reply:
This is a mess, but ok.